include/tc/core/cuda/cuda_tc_executor.h

```diff
@@ -23,20 +28,30 @@
 #include "tc/core/utils/dlpack.h"
 #include "tc/lang/parser.h"
 
-#include <dlpack/dlpack.h>
-
 namespace tc {
 
 class CudaTcExecutor : public ::tc::TcExecutor {
  public:
   CudaTcExecutor(
-      const std::string& def,
-      const std::vector<const DLTensor*>& inputsInfo)
-      : TcExecutor(def, inputsInfo) {}
-  CudaTcExecutor(
-      lang::TreeRef tree,
-      const std::vector<const DLTensor*>& inputsInfo)
-      : TcExecutor(tree, inputsInfo) {}
+      std::string id,
+      std::vector<const DLTensor*> inputsInfo,
```

Comment:
Why did you change the type of the first two argument?
That is, why are they no longer const references?
Note that the second argument of the `TcExecutor` constructor
_is_ a const reference.

Reply:
Didn't change the type of the first 2 arguments.
This is a consolidation of 2 constructors of class (Cuda)TcExecutor  + 1 constructor of class (Cuda)ExecutorInfo into a single constructor of a (Cuda)TcExecutor class.
The signature of the resulting constructor is what makes sense for the consolidated class.

Reply:
Ah yes I missed the point about const, addressing.